app/pages/lab-pages-editor/components/TasksPage/components/EditStepDialog/types/DrawingTask.jsx

```diff
@@ -0,0 +1,347 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import DeleteIcon from '../../../../../icons/DeleteIcon.jsx';
+import DrawingToolIcon from '../../../../../icons/DrawingToolIcon.jsx';
+import MinusIcon from '../../../../../icons/MinusIcon.jsx';
+import PlusIcon from '../../../../../icons/PlusIcon.jsx';
+
+const DEFAULT_HANDLER = () => {};
+
+const TOOL_COLOR_OPTIONS = [
+  { value: '#ff0000', label: 'Red' },
+  { value: '#ffff00', label: 'Yellow' },
+  { value: '#00ff00', label: 'Green' },
+  { value: '#00ffff', label: 'Cyan' },
+  { value: '#0000ff', label: 'Blue' },
+  { value: '#ff00ff', label: 'Magenta' }
+];
```

Comment:
I'd consider inverting this as an Object so it's more useful.. For example:

`TOOL_COLOR_OPTIONS = {
  'Red': '#ff0000'
}`

Then, when you need a color you're able to use the name of the color instead of searching through an array to find a hex value. It makes your loop through a little different (Object.keys(TOOL_COLOR_OPTIONS).map) but from a readability perspective it's a little more clear. It also creates the opportunity to abstract this bit of code out and up for anywhere we want color selection like this.

Not a dealbreaker but a thought to consider.

Reply:
😆 I actually structured it as an object first, before turning it into an array!

I can't remember 100% why I changed it to an array, but I think it was following a discussion with Sam & Sean about 1. the choice of colours we want to present to users, and 2. the name of those colours (e.g. "#89e097 apple" instead of "#00ff00 green"). (I probably just left this as an array until we sorted out that question. 🤔 )

Bonus trivia: the following are some random half-remembered bits from the discussion, just for interest:
- The PFE colour choices are extremely garish and not part of any branding...
- ...but garish colours happen to provide _excellent contrast_ for marks drawn on top of camera trap photos and documents.
- Nobody knows why #00ff00 green is the default.
- I think we could technically allow users to set any colour they want, since Panoptes just stores the hex code and both PFE & FEM classifiers renders any valid hex colour, but we wanted to make this selection as simple as possible for project owners.